fxgl-core/src/main/kotlin/com/almasb/fxgl/core/math/PerlinNoiseGenerator.kt

```diff
@@ -90,6 +95,6 @@ internal object PerlinNoiseGenerator {
 
         // Modulate with the weight function
         val wx = (3 - 2 * tx0) * tx0 * tx0
-        return v0 - wx * (v0 - v1)
+        return v0 - wx * (v0 - v1) + 0.5
```

Comment:
Please check that the behaviour remains unchanged for `FXGLMath.noise1D(t)`

Reply:
Yes behavior is unchanged. Of course now if someone has a game with specific settings based on the old `noise1D` pattern, it will be entirely different with this updated API.